server/chat/backend/agent/tools/cloud_exec_tool.py

```
@@ -1254,7 +1272,215 @@ def _run_on_account(conn: dict) -> dict:
+            cmd = f"gcloud {cmd}"
+        if cmd.startswith("gcloud") and effective_project and "--project" not in cmd \
+                and "gcloud config" not in cmd:
+            cmd += f" --project={effective_project}"
+
+        result = terminal_run(
+            shlex.split(cmd), capture_output=True, text=True,
+            timeout=get_command_timeout(cmd, timeout), env=isolated_env,
+        )
+        ok_run = result.returncode == 0
+        output = (result.stdout if ok_run else (result.stderr or result.stdout)).strip()
+        return ok_run, f"[project: {effective_project} / sa: {alias_or_email}]\n{output}"
```

Ruff (0.15.14)

[warning] Consider moving this statement to an `else` block

(TRY300)